src/seedu/addressbook/Main.java

```diff
@@ -117,6 +115,30 @@ private CommandResult executeCommand(Command command)  {
         }
     }
 
+    /**
+     * Show the list of contacts in addressbook after add or delete command
+     * @param userCommandText
+     */
+
+    private void showListAfterAddOrDelete(String userCommandText) {
+        if (commandIsAddOrDelete(userCommandText)) {
+            CommandResult result = executeCommand(new ListCommand());
+            recordResult(result);
+            ui.showResultToUser(result);
+        }
+    }
+
+    /**
+     * Checks if user command is add or delete
+     * @param userCommandText
```

Comment:
For header comment there should be a blank line between description and `@param`.